Makefile

```diff
@@ -257,15 +255,11 @@ app/controllers.go: $(DESIGNS) $(GOAGEN_BIN) $(VENDOR_DIR)
 	$(GOAGEN_BIN) client -d ${PACKAGE_NAME}/${DESIGN_DIR}
 	$(GOAGEN_BIN) swagger -d ${PACKAGE_NAME}/${DESIGN_DIR}
 	$(GOAGEN_BIN) client -d github.com/fabric8-services/fabric8-tenant/design --notool --pkg tenant -o account
+	$(GOAGEN_BIN) client -d github.com/fabric8-services/fabric8-tenant/design --notool --pkg tenant -o vendor/github.com/fabric8-services/fabric8-auth/account
```

Comment:
Is this indented?

Reply:
yes, this is needed, otherwise the build will fail: the `fabric8-auth/account/tenant` is needed during the compilation, but its content must be generated with `goagen` since it's not on GitHub: https://github.com/fabric8-services/fabric8-auth/tree/master/account